ui_components/src/lib/components/input/InputComponent.svelte

```diff
@@ -276,34 +305,61 @@
                 </div>
             {/if}
 
+            <!-- Required switch -->
             <div class="form-check form-switch mb-3">
                 <label class="form-label">
                     Required
                     <input type="checkbox" class="form-check-input" role="switch" bind:checked={config.required}/>
                 </label>
             </div>
 
+            {#if canDisableFields }
+            <!-- Disabled switch -->
+            <div class="form-check form-switch mb-3">
+                <label class="form-label">
+                    Disabled
+                    <input type="checkbox" class="form-check-input" role="switch" bind:checked={config.disabled}/>
+                </label>
+            </div>
+            {/if}
+
             <div>
-                <button class="btn btn-primary" onclick={() => {onDuplicateRequest()}}><i class="bx bx-duplicate"></i> Duplicate</button>
+                <button class="btn btn-primary" onclick={() => {onDuplicateRequest()}} title="Copy this field">
+                    <i class="bx bx-duplicate"></i> Duplicate
```

Comment:
Disable this if readonly?

Reply:
As we identify questions by the text title, I was worried about the possibility of repeat fields but we can just filter by the `readonly` property so I guess it's not an issue.